src/dmclock_client.h

```diff
@@ -122,9 +122,12 @@ namespace crimson {
 	} else {
 	  Counter delta =
 	    delta_counter - it->second.delta_prev_req;
+#ifdef USE_SEPARATED_RHO_CAL
```

Comment:
Can you explain the motivation for this change? I didn't understand from your write-up.